bigquery_etl/format_sql/tokenizer.py

```diff
@@ -352,9 +682,34 @@ class AngleBracketKeyword(ReservedKeyword):
 
 
 class Identifier(Token):
-    """Token that identifies a column, parameter, table, or other database object."""
+    """Identifier for a column, table, or other database object."""
+
+    pattern = re.compile(r"[A-Za-z_][A-Za-z_0-9]*|`(?:\\.|[^\\`])+`")
+
+
+class QualifiedIdentifier(Identifier):
+    """Fully or partially qualified identifier for a column, table, or other database object."""
+
+    pattern = re.compile(
+        rf"(?:(?:{Identifier.pattern.pattern})\.)+(?:{Identifier.pattern.pattern})"
+    )
```

Comment:
Qualified identifiers are now tokenized as a single token so that qualified references to UDFs with the same name as a built-in function won't get capitalized (e.g. `mozfun.map.sum()`).